docs/packaging.rst

```diff
@@ -2,5 +2,89 @@
 Packaging Guide
 ===============
 
-Documentation can be found at
-https://github.com/letsencrypt/letsencrypt/wiki/Packaging.
+Releases
+========
+
+We release packages and upload them to PyPI (wheels and source tarballs).
+
+- https://pypi.python.org/pypi/acme
+- https://pypi.python.org/pypi/letsencrypt
+- https://pypi.python.org/pypi/letsencrypt-apache
+- https://pypi.python.org/pypi/letsencrypt-nginx
+- https://pypi.python.org/pypi/letshelp-letsencrypt
+
+The following scripts are used in the process:
+
+- https://github.com/letsencrypt/letsencrypt/blob/master/tools/release.sh
+- https://gist.github.com/kuba/b9a3a2ca3bd35b8368ef
+
+We currently version as ``0.0.0.devYYYYMMDD``, and will change at GA time to the following scheme:
+
+- ``0.1.0``
+- ``0.2.0dev`` for developement in ``master``
+- ``0.2.0`` (only temporarily in ``master``)
+- ...
+
+Tracking issue for non-dev release scripts: https://github.com/letsencrypt/letsencrypt/issues/1185
+
+Notes for package maintainers
+=============================
+
+0. Please use our releases, not ``master``!
+
+1. Do not package ``letsencrypt-compatibility-test`` - it's only used internally.
+
+2. ``letsencrypt-renewer`` should be added to crontab... but it currently doesn't work well
+
+3. ``letsencrypt.client`` provides developer API so it should be possible to ``import letsencrypt.client`` when the package is installed (``letsencrypt`` vs ``python-letsencrypt`` debate for Debian).
+
+4. ``jws`` is an internal script for ``acme`` module and it doesn't have to be packaged - it's mostly for debugging: you can use it as ``echo foo | jws sign | jws verify``.
+
+5. Do get in touch with us. We are happy to make any changes that will make packaging easier. If you need to apply some patches don't do it downstream - make a PR here.
+
+Already ongoing efforts
+=======================
```

Comment:
Many of these links are outdated and a number of packages are missing. Ideally, we should add all known packages here (using https://certbot.eff.org as a reference) with updated links.